### PR TITLE
UI: Reconnect network page imporved.

### LINKED ISSUE #862: Improve reconnect error page.
The visual appearance of the reconnect error page we see when there are problems with the internet connection can be significantly improved.

Screenshot from 2020-01-14 18-48-18

- **Clear steps to reproduce the issue**:
Start the app without an internet connection

### PR DESCRIPTION
**Pull request by samyakgaur**
**What's this PR do?**
Improves the design alignment of network reconnect page.

**Any background context you want to provide?**
Fixes #862 

**Screenshots?**
Screenshot 2020-07-08 at 1 19 03 PM

**You have tested this PR on:**
  - [ ] Windows
  - [ ] Linux/Ubuntu
  - [x] macOS

### CODE REVIEW

**Comment by manavmehta** [comment]
Checkout comments on #872. Try to find some graphics or maybe change the background color - a darker colour which could say there is connectivity error !?!

**Comment by samyakgaur** [comment]
> Checkout comments on #872. Try to find some graphics or maybe change the background color - a darker colour which could say there is connectivity error !?!

Yeah i had read it. should i make the background darker ? because using any random image from internet could lead to copyright issue that is the reason i preferred using the old one.

**Comment by akashnimare** [comment]
@samyakgaur before opening a PR please discuss the approach, UI changes etc with the community on [czo](https://chat.zulip.org/#narrow/stream/16-desktop). This will save a lot of your time.

**Comment by samyakgaur** [comment]
> @samyakgaur before opening a PR please discuss the approach, UI changes etc with the community on [czo](https://chat.zulip.org/#narrow/stream/16-desktop). This will save a lot of your time.

Yes @akashnimare i had done that but didn’t get any reply so i moved forward with this approach. Not an issue ill ask once again :)

File: app/renderer/css/network.css [hunk 1]
```diff
@@ -44,16 +81,56 @@ body {
...
+    position: relative;
+}
+
+.cog {
+    margin-top: 50px;
+    width: 10rem;
+    height: 10rem;
+    animation: coganimation 10s infinite;
+    fill: rgba(0, 150, 136, 1.000);
+    margin-left: 60px;;
```
Comment (kanishk98) [inline app/renderer/css/network.css]: Could we please remove the extra `;` here? @manavmehta I guess our linters miss this sort of thing?

File: app/renderer/network.html [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 <html lang="en" class="responsive desktop">
-    <head>
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-        <meta name="viewport" content="width=device-width">
-        <title>Zulip - Network Troubleshooting</title>
-        <link rel="stylesheet" href="css/network.css" type="text/css" media="screen">
-    </head>
-    <body>
+
+<head>
```
Comment (kanishk98) [inline app/renderer/network.html]: Please indent the `<head>` block. If you look at the diff, you should notice a difference.

**Review by kanishk98 — CHANGES_REQUESTED** [review]
Thanks for the PR, @samyakgaur! Left a couple of nitpicky comments, nothing major - but please do follow @akashnimare's advice and save everyone a lot of time by discussing things before you implement them.

File: app/renderer/network.html (hunk 2 above)
Comment (samyakgaur) [reply app/renderer/network.html]: Tried doing this, now linting tests are failing. I tried going back to the old code version but linting fails there also. I guess tabs are not being considered as 4 spaces if i'm not wrong

File: app/renderer/css/network.css (hunk 1 above)
Comment (manavmehta) [reply app/renderer/css/network.css]: @kanishk98 Thanks for pointing out! I'll definitely look into it

File: app/renderer/network.html [hunk 3]
```diff
@@ -1,26 +1,34 @@
...
+    	        </div>
+        	    <ul id="description">
+            	    <li>You're not online or your proxy is misconfigured.</li>
+                	<li>There is no Zulip organization hosted at this URL.</li>
+	                <li>This Zulip organization is temporarily unavailable.</li>
+    	            <li>This Zulip organization has been moved or deleted.</li>
+	            </ul>
+    	        <div id="buttons">
+        	        <div id="reconnect" class="button">Reconnect</div>
+            	    <div id="settings" class="button">Settings</div>
```
Comment (akashnimare) [inline app/renderer/network.html]: @samyakgaur do you understand what's going on here?

Comment (samyakgaur) [reply app/renderer/network.html]: @akashnimare Yeah we will have to link it to the respective page. Or should i remove it ?

Comment (akashnimare) [reply app/renderer/network.html]: Ideally, we should be using button tag here -
```
<div class="button-container">
<button id="reconnect">Reconnect</button>
<button id="settings">Settings</button>
</div>
```

**Comment by akashnimare** [comment]
@samyakgaur are you up for making the above changes? Once done, we can merge.

**Comment by samyakgaur** [comment]
> @samyakgaur are you up for making the above changes? Once done, we can merge.

@akashnimare sorry, almost forgot about this PR. will make the following changes and commit it.

**Pull request closed without merging**